src/grpc/clients/ea_restaurant_java_etl_grpc_client.py

```diff
@@ -18,14 +26,19 @@ def __init__(self, grpc_client):
                 self.channel
             )
         )
+        self.env_config = get_env_config_instance()
+        self.access_token = None
+        self.refresh_token = None
+        self.first_time_login = False
+        self.mongo_uuids = None
 
-    def login_client(self, client_id, client_secret):
+    def login_client_call(self, client_id, client_secret):
         basic_token = generate_basic_token_from_credentials(client_id, client_secret)
         metadata = (("authorization", basic_token),)
         no_param = java_etl_grpc_client_pb2.NotParametersRequest()
```

Comment:
Maybe `no_param_req` instead of `no_param`

Reply:
Done.